install-osx.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh
 echo 'Installing .NET Core...'
 
-wget https://download.visualstudio.microsoft.com/download/pr/4850aa8f-44a9-4c4a-9961-f18aa4d90ceb/07d790444f3ba6b412a76d6f1aced338/dotnet-sdk-2.2.105-osx-x64.pkg -O ~/dotnet-sdk-2.2.105-osx-x64.pkg
+wget https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-osx-x64.pkg -O ~/dotnet-runtime-2.0.9-osx-x64.pkg
```

Comment:
@brendanburns did you remove travis from this repo?
I did not see it triggered anymore

I mean we can remove those files for travis if no longer used